extruct/schemaorg.py

```diff
@@ -0,0 +1,31 @@
+"""Schema.org function helpers."""
```

Comment:
It looks like this module, as well as  extruct.transform are not needed if we remove transformations from the PR.